src/commcare_cloud/ansible/plugins/lookup/cchq_aws_secret.py

```diff
@@ -38,6 +38,7 @@ def run(self, terms, variables, **kwargs):
             if value is Ellipsis:
                 logging.debug('Fetching secret {}'.format(terms))
                 try:
+                    # ----- Here we need to get the environment variable ----- #
```

Comment:
If you first make sure 'aws_region' is in `public.yml` (which it is on staging) can you use `region = variables['aws_region']` here?

Reply:
@dannyroberts yes, this approach is also working. And as discussed, we will go with `os.environ['AWS_REGION']`